benchmarking/collect_LIBSVM.sh

```diff
@@ -18,21 +18,18 @@
 # and working.
 
 
-if [[ "$#" != 2 ]]; then
-    echo "Usage: collect_LIBSVM.sh temporary_dir output_dir" 1>&2
+if [[ "$#" != 1 ]]; then
+    echo "Usage: collect_LIBSVM.sh output_dir" 1>&2
     exit 1
 fi
 
-TEMP_DIR="$1"
-DEST_DIR="$2"
+DEST_DIR="$1"
 
-mkdir -p "${TEMP_DIR}" || exit 1
 mkdir -p "${DEST_DIR}" || exit 1
 
 # To download the binary problems:
 data_source="https://www.csie.ntu.edu.tw/~cjlin/libsvmtools/datasets/binary"
 for filename in kdda.t real-sim avazu-app.val; do
-    curl "${data_source}/${filename}.bz2" --output "${TEMP_DIR}/${filename}.bz2" || exit 1
-    bunzip2 -d "${TEMP_DIR}/${filename}.bz2"
-    mv "${TEMP_DIR}/${filename}" "${DEST_DIR}/${filename}"
+    curl "${data_source}/${filename}.bz2" --output "${DEST_DIR}/${filename}.bz2" || exit 1
```

Comment:
Given that we're simplifying, what about just
    curl "${data_source/${filename}.bz2" | bzcat > "${DEST_DIR}/${filename}"

Reply:
This would suggest using PIPESTATUS for error handling, no? There might be a compact way to do it, but to me it's not obviously a net simplification.

Reply:
Checking PIPESTATUS would only be necessary to avoid additional commands or perform cleanup if there was a failure, or to adjust the error messaging.  In this case, the "normal" error would probably be enough.  But it's not a major simplification so lgtm.